src/main/java/com/kodedu/controller/ApplicationController.java

```diff
@@ -2830,16 +2831,22 @@ public void createFolder(ActionEvent actionEvent) {
 
             if (result.matches(DialogBuilder.FOLDER_NAME_REGEX)) {
 
-                Path path = fileSystemView.getSelectionModel().getSelectedItem()
-                        .getValue().getPath();
+                Path path = null;
```

Comment:
Below three code changes seems identical, can we exclude them to a single method ?

Reply:
Done